units/patch_intent.go

```diff
@@ -914,15 +914,15 @@ func (j *patchIntentProcessor) buildTriggerPatchDoc(patchDoc *patch.Patch) error
 	return nil
 }
 
-func (j *patchIntentProcessor) verifyValidAlias(projectId string, patchDoc *patch.Patch) error {
+func (j *patchIntentProcessor) verifyValidAlias(projectId string, configStr string) error {
 	alias := j.intent.GetAlias()
 	if alias == "" {
 		return nil
 	}
 	var projectConfig *model.ProjectConfig
-	if patchDoc.PatchedProjectConfig != "" {
+	if configStr != "" {
 		var err error
```

Comment:
This line can be removed

Reply:
It can't be because projectConfig needs to be used outside of the if statement